### DISCUSSION TITLE
2023년 06월 03일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#59] [General]
# 2023년 06월 03일

LOG 날씨가 매우 좋았는데, 동시에 더웠다... 집에서 에어컨 틀고 누워있으니 기분이 진짜 좋았다. 오랜만에 집에서 김치찌개 해먹었는데 역시 맛있었다. 미루던 집안일도 해결한 오늘의 점수 :100: 배운 것 헤나가 말하기를 sql 버전에 따라 al…

https://vero.wiki/2023-06-03/

**Comment by Choi-JJunho** [comment]
방학에는 노세요~

**Reply by Cyma-s** [reply]
그치만 알고리즘이랑 CS는 해야 하는 걸...
